### PR TITLE
Create yml file to handle linting with github actions

### PR DESCRIPTION
**Pull request by afroman32**
# Description
This pull request utilizes github actions to check python files that have been changed for syntax errors and for formatting standards. Files that have errors will be flagged and the necessary changes will be noted. Formatting suggestions will be available as well.
This test will only run when code is being pushed, and will only check files that have been edited or created since the last push.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Name the job
```
Comment (ike-okonkwo) [inline .github/workflows/linter.yml:1]: @afroman32   lets add a readme to this PR as well

**Review by ike-okonkwo — DISMISSED** [review]

File: .github/workflows/linter.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+name: Linting
```
Comment (peterger8y) [inline .github/workflows/linter.yml:2]: Fascinating. is this Github tech?

**Review by peterger8y — APPROVED** [review]

**Review by ike-okonkwo — APPROVED** [review]
Thanks for updating the readme and starting the foundational work for the CI / CD  pipeline

File: .github/workflows/linter.yml (hunk 2 above)
Comment (afroman32) [reply .github/workflows/linter.yml:2]: yes it is

**Pull request merged**